banzai/calibrations.py

```diff
@@ -45,7 +45,7 @@ def run(self, images):
         return processed_images
 
     def do_stage(self, images):
-        min_images = self.pipeline_context.CALIBRATION_MIN_IMAGES.get(self.calibration_type.upper(), -1)
+        min_images = self.pipeline_context.CALIBRATION_MIN_FRAMES.get(self.calibration_type.upper(), -1)
```

Comment:
why is that -1 and not 0?

Reply:
It was due to the possibility of there being 0 images, however I've refactored this section to give a more specific error in case the key is missing.